service/src/test/java/bio/terra/workspace/pact/TpsApiTest.java

```diff
@@ -35,6 +34,8 @@
 @Tag("pact-test")
 @ExtendWith(PactConsumerTestExt.class)
 public class TpsApiTest {
+  private static final String UUID_REGEX =
```

Comment:
As part of upgrading the pactjvm dep, we lost access to the `UUID_REGEX` which was buried in an inaccessible kotlin companion object.  This is just an inline of the String previously referenced.